pwiz_tools/Skyline/Controls/Graphs/GraphFullScan.cs

```diff
@@ -600,6 +612,8 @@ public override int GetHashCode()
                         _transitionIndex = new int[]{};
                 }
 
+                var isHighRes = !Equals(settings.TransitionSettings.FullScan.ProductMassAnalyzer,FullScanMassAnalyzerType.qit);

```

Comment:
I think this line is no longer necessary.

Reply:
Right. I wonder how it passed code inspection.